client/blocks/upsell-nudge/style.scss

```diff
@@ -0,0 +1,52 @@
+.upsell-nudge {
+	border-left: 8px solid var( --color-accent );
+	background-color: var( --color-neutral-80 );
+	color: var( --color-text-inverted );
+	display: flex;
+	align-items: center;
+	margin-bottom: 1.5em;
+	padding: 0.5em 1.5em;
+
+	a {
+		color: var( --color-text-inverted );
+	}
+
+	&__content {
+		padding: 1em 1em 1em 0;
+		margin-right: auto;
+	}
+
+	&__dismiss {
+		margin-left: 1em;
+	}
+
+	&__dismiss-icon {
+		fill: var( --color-text-inverted );
+	}
+
+	&__button {
+		margin-left: auto;
+		overflow: visible;
+		white-space: nowrap;
+	}
+
+	&.compact {
```

Comment:
We have a convention (the BEM system, although slightly modified) to use a `.is-` prefix for modifier classes like this: `.is-compact`. ([Standard BEM](http://getbem.com/naming/) would use `.upsell-nudge--compact`)